src/components/RichTextEditor/Plugins/ListPlugin/ListItemContentMarkupElement.tsx

```diff
@@ -25,7 +25,7 @@ export const getUnderlineClassNames = (designTokens: DesignTokens, element: TEle
 export const getLicElementClassNames = (element: TElement) =>
     merge([
         getColumnBreakClasses(element),
-        element.align ? justifyClassNames[element.align as string] : 'tw-justify-start tw-flex',
+        element.align ? justifyClassNames[element.align as string] : 'tw-justify-start',
```

Comment:
Removed `tw-flex`, because we have `tw-grid` on the next line anyway and there both don't go together :)